compiler/bindings/python/iree/compiler/tools/tflite.py

```diff
@@ -30,6 +32,11 @@
 
 def is_available():
     """Determine if the TFLite frontend is available."""
+    try:
+        import tensorflow as tf
+    except ModuleNotFoundError:
+        logging.warn("Unable to import tensorflow")
+        return False
     try:
         import iree.tools.tflite.scripts.iree_import_tflite.__main__
```

Comment:
> The weird thing is that the tests in https://github.com/iree-org/iree/blob/main/compiler/bindings/python/test/tools/compiler_tflite_test.py should have exposed the bugs. When I ran them locally most of them failed before. Are they not run on CI? @ScottTodd

I see the tests running: https://github.com/iree-org/iree/actions/runs/8929912904/job/24528824741#step:5:1403

but they might be getting skipped: https://github.com/iree-org/iree/blob/71a994559068929f206ca60b764416ebfe65daa2/compiler/bindings/python/test/tools/compiler_tflite_test.py#L17-L22

Reply:
The base compiler build does not include large Python deps like TensorFlow. This other job is running some TF tests: https://github.com/iree-org/iree/blob/71a994559068929f206ca60b764416ebfe65daa2/.github/workflows/pkgci_test_tensorflow_cpu.yml#L47-L50

We could update https://github.com/iree-org/iree/blob/main/build_tools/cmake/run_tf_tests.sh to run these compiler/bindings/python/ tests too... 

https://github.com/iree-org/iree/issues/16624 is a similar issue for another test in that directory

Reply:
I added this to the file and it seems like they run now.